chainer_chemistry/datasets/reddit/.gitignore

```diff
@@ -0,0 +1,3 @@
+reddit.zip
```

Comment:
remove this file. I think dataset should be inside `examples` directory, not inside library.

Reply:
fixed!